blueocean-dashboard/src/main/js/OrganizationPipelines.jsx

```diff
@@ -140,15 +147,15 @@ OrganizationPipelines.propTypes = {
     params: object, // From react-router
     children: node, // From react-router
     location: object, // From react-router
-    pipelines: array,
+    allPipelines: array,
+    organizationPipelines: array,
 };
 
 OrganizationPipelines.childContextTypes = {
     pipelines: array,
-    pipeline: object,
 };
 
-const selectors = createSelector([pipelinesSelector],
-    (pipelines) => ({ pipelines }));
+const selectors = createSelector([allPipelinesSelector, organizationPipelinesSelector],
+    (allPipelines, organizationPipelines) => ({ allPipelines, organizationPipelines }));
```

Comment:
how come this is now split in twain, org pipelines vs all? (I remember some discussion about it, just not sure how it relates to this... remind me?)